docker/templates/django/{{cookiecutter.app_name}}/{{cookiecutter.module_name}}/settings.py

```diff
@@ -0,0 +1,161 @@
+"""
+Django settings for example_app project.
+
+Generated by 'django-admin startproject' using Django 3.0.3.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.0/ref/settings/
+"""
+import os
+import socket
+
+from django.core.exceptions import ImproperlyConfigured
+import dj_database_url
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+# Retrieve the secret key from the DJANGO_SECRET_KEY environment variable
+SECRET_KEY = os.environ['DJANGO_SECRET_KEY']
+
+# Set the DJANGO_DEBUG environment variable to False to disable debug mode
+DEBUG = False if os.getenv('DJANGO_DEBUG', True) == 'False' else True
+
+# Define DJANGO_ALLOWED_HOSTS as a comma-separated list of valid hosts,
+# e.g. localhost,127.0.0.1,.herokuapp.com
+allowed_hosts = os.getenv('DJANGO_ALLOWED_HOSTS', [])
+ALLOWED_HOSTS = allowed_hosts.split(',') if allowed_hosts else []
+
```

Comment:
We should initialize Sentry here, too! h/t @beamalsky

Reply:
Great thinking, added! I also went ahead and implemented the 400 request grouping we recommended in https://github.com/datamade/devops/issues/111, thanks for mentioning that during DevOps.